Tryouts/Plugins/Services/MockAuthentication/MockAuthenticationService.cs

```diff
@@ -0,0 +1,75 @@
+﻿/// ********************************************************************************************************
+///
+/// Morgan Stanley makes this available to you under the Apache License, Version 2.0 (the "License").
+/// You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+/// See the NOTICE file distributed with this work for additional information regarding copyright ownership.
+/// Unless required by applicable law or agreed to in writing, software distributed under the License
+/// is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and limitations under the License.
+/// 
+/// ********************************************************************************************************
+
+using NP.Utilities;
+using NP.Utilities.Attributes;
+using NP.Utilities.BasicServices;
+
+namespace MockAuthentication;
+
+[Implements(typeof(IAuthenticationService), IsSingleton = true)]
+public class MockAuthenticationService : VMBase, IAuthenticationService
```

Comment:
VMBase is = view model base, something that container INotifiPropertyChanged.PropertyChanged event and OnPropertyChanged(string propName) method to be use to signify the property changes from the bindings.

Reply:
I think I wasn't clear in my question because I was inferring some things.  I could have said "I assume VMBase means ViewModelBase.  In the world of MVVM (and Services), ViewModel is distinct from Service.  A ViewModels responsibility is tightly aligned with a View, even if they aren't directly coupled in the direction of VM->V.  A Services responsibility is entirely decoupled from the notion of any particular View.  Seeing a Service that "is-a" ViewModel creates cognitive dissonance and I feel that it is a code smell.  What value does doing so provide here, and does the benefit it provide outweigh the cost of the cognitive dissonance?"

Reply:
This has been removed from the current stage of the code.